dist/jquery.jsonview.js

```diff
@@ -250,11 +252,13 @@ Licensed under the MIT License.
       defaultOptions = {
         collapsed: false,
         nl2br: false,
-        recursive_collapser: false
+        recursive_collapser: false,
+        enableHTMLKeys: false
```

Comment:
I'd like to keep options names simple, mayby just call this option `escape` and set it's default to true.